### PR TITLE
Version Autoupdate

### PR DESCRIPTION
**Pull request by planemo-autoupdate**
Autoupdate conda package versions

### CODE REVIEW

**Comment by bgruening** [comment]
@simonbray do you think we can create a PR per tool?

File: chemicaltoolbox/rdock/rdock_macros.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <macros>
-    <token name="@TOOL_VERSION@">2013.1-0</token>
+    <token name="@TOOL_VERSION@">2013.1</token>
```
Comment (bgruening) [inline chemicaltoolbox/rdock/rdock_macros.xml:2]: Oh that is an interesting one ....

File: chemicaltoolbox/rdock/select_points_SDF.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@
         <token name="@GALAXY_VERSION@">0</token>
     </macros>
     <requirements>
-        <requirement type="package" version="3.7">python</requirement>
+        <requirement type="package" version="3.9.1">python</requirement>
```
Comment (bgruening) [inline chemicaltoolbox/rdock/select_points_SDF.xml:8]: This is discussable, maby we should skip python and r-base upgrades?

File: tools/mafft/mafft.xml [hunk 3]
```diff
@@ -1,294 +1,294 @@
...
-      --thread
-      Number of threads (if unsure, --thread -1)
-      --retree number
-      Guide tree is built number times in the progressive stage.
-      Valid with 6mer distance.  Default: 2
-      ]]>
-    </help>
-  <expand macro="citations" />
-</tool>
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (bgruening) [inline tools/mafft/mafft.xml:1]: I assume some lineending changes here?

**Comment by simonbray** [comment]
I fixed the line-ending and python issues in two new commits here: https://github.com/galaxyproject/planemo/pull/1065

File: chemicaltoolbox/rdock/rdock_macros.xml (hunk 1 above)
Comment (simonbray) [reply chemicaltoolbox/rdock/rdock_macros.xml:2]: Hm, do you know what the `-0` means here?

**Comment by bgruening** [comment]
Cool thanks @simonbray!

**Pull request closed without merging**